packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -401,6 +411,48 @@ async function updateAutoLink(
         changesNecessary;
     }
 
+    // Generating props for app project consumption
+    let propertiesForProps = '';
+    let addCsModuleSupport: boolean = false;
+
+    if (projectLang === 'cpp') {
+      for (const dependencyName in windowsDependencies) {
+        windowsDependencies[dependencyName].projects.forEach(project => {
+          if (project.directDependency && project.projectLang === 'cs') {
+            addCsModuleSupport = true;
+          }
+        });
+      }
+
+      if (addCsModuleSupport) {
+        propertiesForProps += `\n    <ConsumeCSharpModules>true</ConsumeCSharpModules>`;
+      }
```

Comment:
do we need to check whether this prop is already defined?

Reply:
Why would we support that? So people could break their build?